security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -15,6 +15,24 @@ For an overview of the File Integrity Operator, see xref:../../security/file_int
 
 To access the latest release, see xref:../../security/file_integrity_operator/file-integrity-operator-updating.adoc#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-1-3-6"]
+== OpenShift File Integrity Operator 1.3.6
+ 
+The following advisory is available for the OpenShift File Integrity Operator 1.3.6:
+
+* link:https://access.redhat.com/errata/RH??-2025:????[RH??-2025:???? OpenShift File Integrity Operator Bug Fix Update]
+
+[id="file-integrity-operator-1-3-6-bug-fixes"]
+=== Bug fixes
+
+* Previously, FIO would re-initialize on failure for all nodes in the cluster. With this release, FIO has been updated to re-initialize the database on failure nodes only.  (link:https://issues.redhat.com/browse/OCPBUGS-18933[*OCPBUGS-18933*])
```

Comment:
For additional clarity here, FIO won't automatically re-initialize databases, but let's users re-init a database to clear the state and start fresh.

In this particular fix, the logic to reinit is now only being applied when a user asks FIO to clear state.